Switcheroo/MainWindow.xaml.cs

```diff
@@ -109,6 +109,76 @@ private void SetUpKeyBindings()
                     tb.IsEnabled = true;
                     tb.Focus();
                 }
+                else if (args.SystemKey == Key.D1 && Keyboard.Modifiers.HasFlag(ModifierKeys.Alt))
+                {
+                    lb.SelectedIndex = 0;
+                    ScrollSelectedItemIntoView();
+                    Switch();
+                    HideWindow();
+                }
+                else if (args.SystemKey == Key.D2 && Keyboard.Modifiers.HasFlag(ModifierKeys.Alt))
+                {
+                    lb.SelectedIndex = 1;
+                    ScrollSelectedItemIntoView();
+                    Switch();
+                    HideWindow();
+                }
+                else if (args.SystemKey == Key.D3 && Keyboard.Modifiers.HasFlag(ModifierKeys.Alt))
+                {
+                    lb.SelectedIndex = 2;
+                    ScrollSelectedItemIntoView();
+                    Switch();
+                    HideWindow();
+                }
+                else if (args.SystemKey == Key.D4 && Keyboard.Modifiers.HasFlag(ModifierKeys.Alt))
+                {
+                    lb.SelectedIndex = 3;
+                    ScrollSelectedItemIntoView();
+                    Switch();
+                    HideWindow();
+                }
+                else if (args.SystemKey == Key.D5 && Keyboard.Modifiers.HasFlag(ModifierKeys.Alt))
+                {
+                    lb.SelectedIndex = 4;
+                    ScrollSelectedItemIntoView();
+                    Switch();
+                    HideWindow();
+                }
+                else if (args.SystemKey == Key.D6 && Keyboard.Modifiers.HasFlag(ModifierKeys.Alt))
+                {
+                    lb.SelectedIndex = 5;

```

Comment:
When pressing `Alt+5` but no only 3 windows are available then it switches to the first window in the list. I think it would be more logical if nothing happened.